Dockerfile

```diff
@@ -1,11 +1,18 @@
+FROM golang:1.13.4-alpine as build
+RUN mkdir -p /go/src/github.com/aosapps/drone-sonar-plugin
+WORKDIR /go/src/github.com/aosapps/drone-sonar-plugin 
+COPY *.go ./
```

Comment:
I think it's better to copy all files. This way it has the same behaviour as building it locally.

Reply:
If you prefer. If it's too many useless files at some point, you can use a `.dockerignore` file.